packages/react-magma-dom/src/components/AlertBase/index.tsx

```diff
@@ -40,6 +38,7 @@ export interface AlertBaseProps extends React.HTMLAttributes<HTMLDivElement> {
   closeAriaLabel?: string;
   forceDismiss?: () => void;
   hasTimerRing?: boolean;
+  id?: string;
```

Comment:
Is this used?

Reply:
Why was this prop added?

Reply:
It was missing from the props table and causing an error.